samples/projection-management.ts

```diff
@@ -0,0 +1,363 @@
+import { v4 as uuid } from "uuid";
+
+import { EventStoreDBClient, isCommandError } from "@eventstore/db-client";
+import { createTestNode, delay, jsonTestEvents } from "@test-utils";
+
+describe("[sample] projection-management", () => {
+  const noop = (...args: unknown[]) => {
+    // do nothing
+  };
+  const node = createTestNode();
+  const log = console.log;
+
+  let client!: EventStoreDBClient;
+
+  const createTestProjection = async (
+    name: string = uuid(),
+    retry = 5
+  ): Promise<string> => {
+    try {
+      await client.createContinuousProjection(name, "fromAll().when()");
+    } catch (error) {
+      if (retry > 0) return createTestProjection(name, retry - 1);
+      throw error;
+    }
+
+    return name;
+  };
+
+  beforeAll(async () => {
+    await node.up();
+
+    client = new EventStoreDBClient(
+      { endpoint: node.uri },
+      { rootCertificate: node.rootCertificate },
+      { username: "admin", password: "changeit" }
+    );
+
+    await client.appendToStream("some-stream", jsonTestEvents());
+    console.log = jest.fn(log);
+  });
+
+  afterAll(async () => {
+    console.log = log;
+    await node.down();
+  });
+
+  afterEach(() => {
+    (console.log as jest.Mock).mockReset();
+  });
+
+  test("createClient", async () => {
+    const ENDPOINT = node.uri;
+    const ADMIN = "admin";
+    const PASSWORD = "changeit";
+
+    // region createClient
+    const client = EventStoreDBClient.connectionString`
+        esdb+discover://${ADMIN}:${PASSWORD}@${ENDPOINT}?nodePreference=leader
+    `;
+    // endregion createClient
+
+    noop(client);
+  });
+
+  test("CreateContinuous", async () => {
+    // region CreateContinuous
+
```

Comment:
**FIX:** Remove redundant spaces after `region` (here and in other places). They're rendered in docs and create too much space above the snippets. See e.g.
![obraz](https://user-images.githubusercontent.com/5562528/130410279-fa810d96-0eb9-4215-93d0-f0524b4f6e9d.png)